### DISCUSSION TITLE
Vercel replaces external image domain with deployment domain - NuxtJS

### DISCUSSION THREAD

**Discussion by faruuko** [vercel/community#313] [Help] [answered]
After deploying my nuxt project, vercel chagnes the image source from 

`https://farko.me/uploads/rahc-crsa/originals/f7f5b380-d81e-4ece-9931-05e84e5308ef.jpg` to 
`https://ecowas-rahc.vercel.app/uploads/rahc-crsa/originals/f7f5b380-d81e-4ece-9931-05e84e5308ef.jpg`

Note that the domain changed.

Do you have any idea what's going on?

**Comment by wiesys** [comment, marked as answer]
I had the same problem and couldn't find out what's the problem literally for days. Finally, I came to the conclusion, that the `@nuxtjs/axios` package is making this error. This was a "legacy project", and it was using `@nuxtjs/axios` version 5.12.0.

Still not 100% sure what exactly in axios code was making this error, but **when I updated to the latest (5.13.6), the issue disappeared.** 🥳
